client/rest-high-level/src/test/java/org/elasticsearch/client/RollupIT.java

```diff
@@ -158,5 +166,26 @@ public void testPutRollupJob() throws Exception {
                 }
             }
         });
+
+        // TODO when we move cleaning rollup into ESTestCase we can randomly choose the _all version of this request
+        GetRollupJobRequest getRollupJobRequest = new GetRollupJobRequest(id);
+        GetRollupJobResponse getResponse = execute(getRollupJobRequest, rollupClient::getRollupJob, rollupClient::getRollupJobAsync);
+        assertThat(getResponse.getJobs(), hasSize(1));
+        JobWrapper job = getResponse.getJobs().get(0);
+        assertEquals(putRollupJobRequest.getConfig(), job.getJob());
+        assertThat(job.getStats().getNumPages(), lessThan(10L));
+        assertEquals(numDocs, job.getStats().getNumDocuments());
+        assertThat(job.getStats().getNumInvocations(), lessThan(10L));
+        assertEquals(1, job.getStats().getOutputDocuments());
+        assertEquals(IndexerState.STARTED, job.getStatus().getState());
```

Comment:
Probably need to check for either `STARTED` or `INDEXING` since you could catch the job right after it triggers but before it realizes there's no more data to index